### PR TITLE
Refactor character module and add docstring

### LINKED ISSUE #3: Reduce bloated text before embeds
Right now there are a lot of extra messages in front of the result embed. While potentially useful, they clog up the space too much. Those texts need to be reworded/moved to another location.

### LINKED ISSUE #4: Create help command/page
Currently documentation for how to use the bot is very lacking. A help command or help page is needed.

### PR DESCRIPTION
**Pull request by nnguyen259**
This will close #4 and #3

### CODE REVIEW

**Review by iyZAIRI — DISMISSED** [review]

**Review by elee-github — DISMISSED** [review]

**Comment by nnguyen259** [comment]
Wait for #10 before merge

- [x] Merge #10 first

File: cogs/character.py [hunk 1]
```diff
@@ -208,19 +274,39 @@ async def art(self, ctx, name, version=None):
...
     async def search(self, ctx):
         if ctx.invoked_subcommand is None:
             await ctx.send('No search term specified.')
 
     @char.group()
     @commands.guild_only()
     async def alias(self, ctx):
+        """Manage and view aliases
+
+        Aliases work on a per server basic (i.e Each server keeps its own list of aliases).
```
Comment (iyZAIRI) [inline cogs/character.py]: basis

File: cogs/character.py [hunk 2]
```diff
@@ -208,19 +274,39 @@ async def art(self, ctx, name, version=None):
...
+
+        There are 2 types of aliases: Alternate Name and Version Alias.
+        Alternate Name can be used in place of the character's actual name (i.e Six instead of Seox)
+        Version Alias replace both the name and the version of a character (i.e SKolulu for Summer Kolulu)
```
Comment (iyZAIRI) [inline cogs/character.py]: replaces instead of replace

File: cogs/character.py [hunk 3]
```diff
@@ -208,19 +274,39 @@ async def art(self, ctx, name, version=None):
...
+
+        Both version of aliases will work in place a name and/or version is used, including when adding a new alias.
```
Comment (iyZAIRI) [inline cogs/character.py]: versions

File: cogs/character.py [hunk 4]
```diff
@@ -281,6 +371,13 @@ async def remove(self, ctx, alias):
 
     @alias.command()
     async def list(self, ctx, name, version=None):
+        """Liast alias(es) for the character
```
Comment (iyZAIRI) [inline cogs/character.py]: List

File: cogs/prefix.py [hunk 5]
```diff
@@ -22,19 +23,37 @@ def loadPrefix(self, client, message):
...
             return results
 
     @commands.group()
+    @commands.guild_only()
     async def prefix(self, ctx):
+        """Manage and view prefixes
+
+        The default prefix for KoluluBot is !gbf. This prefix will always work.
+
+        Any additional prefix works on a per server basic (each server has its own prefix list). Multiple prefixes can be active on a server at any given time.
```
Comment (iyZAIRI) [inline cogs/prefix.py]: Basis

**Review by iyZAIRI — DISMISSED** [review]

**Review by elee-github — DISMISSED** [review]

**Pull request merged**